homeassistant/components/media_player/apple_tv.py

```diff
@@ -140,7 +140,7 @@ def media_content_type(self):
             from pyatv import const
             media_type = self._playing.media_type
             if media_type == const.MEDIA_TYPE_VIDEO:
-                return MEDIA_TYPE_VIDEO
+                return MEDIA_TYPE_MOVIE
```

Comment:
**Preference:**
I'd be inclined to keep these as `MEDIA_TYPE_VIDEO` unless the logic specifies movie, as cast does for example, even though it makes this a small breaking change.

Reply:
Especially when it comes to apple I'm usually out. This case isn't any better.
However, while making this change I had a quick look at the constants defined by the used library [here](https://github.com/postlund/pyatv/blob/ecda6df1336bdc3bc530908b85dcd16b608cb8b0/pyatv/const.py#L24-L34).

Considering there is no further type separation between movie/video I think it should be `movie` in this case as well.
But as I'm not going to use the apple tv component I don't really care to much :)

Reply:
Ah, if you've done some upstream digging then I'm good with it!

Reply:
Just because I initially messed up :)
Also, I didn't dig deeper than what I've already mentioned.

As you said, this would be a breaking change and whether or not it makes sense comes down to two things:
1. the majority of content consumed on apple tv devices are *not* actual movies
2. the majority of content *supposed* to be consumed on apple tv devices are not movies

If one of those is true it could/would make sense to change it to `video` but I don't know the answer to any of those so... :)

Reply:
Well it doesn't have to _not_ be a movie. Movies are a subset of videos, so if we don't know that something is specifically a movie, we should just call it a video.

Reply:
Exactly what I mean.
Apple tvs are however built to primarily watch movies. At least that's what I assume so setting the content-type to movie is what I'd expect to happen.

As I said, I'm fine with both though :)

Reply:
I have no strong feelings myself. This PR fixes some messed up constants, so either way is fine with me as well. The question ultimately needs to be answered by a stakeholder who actually uses apple tv, but that doesn't block this PR.

Reply:
The protocol (from ancient iTunes times) supports lots of different media types, you can find some of them in the test:

https://github.com/postlund/pyatv/blob/ecda6df1336bdc3bc530908b85dcd16b608cb8b0/tests/test_convert.py

I decided to group related types together as "super types". So movies, tv shows, music videos, etc. are all reported as "Video". This was mostly to make things easier and I have no intention to change this. If a need for better granularity arises, then I will add a "subtype" instead to not break the API.

Reply:
OK, given this info lets switch this back to `MEDIA_TYPE_VIDEO`.